package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@amcharts/amcharts4": "^4.7.6",
     "@amcharts/amcharts4-geodata": "^4.1.8",
-    "@urkellabs/ucl": "^0.1.31",
+    "@urkellabs/ucl": "^0.1.39",
     "decimal.js": "^10.2.0",
```

Comment:
I'm assuming you know this/have planned for this already but no version 0.1.39 yet for ucl. Breaking the tests.

Reply:
Yeah I think this was a weird issue w/ NPM yesterday. It was deployed, but would only show version 39 on the search page - if you clicked into our repo it said it was only at 38, but then trying to publish 39 would fail as it was "already published". Looks like it finally was registered this morning so going to retry the build here.